guides/doc-Upgrading_and_Updating/topics/cloning_satellite_server.adoc

```diff
@@ -204,7 +205,7 @@ For more information, see {InstallingServerDocURL}configuring-external-services[
 ----
 # {installer-scenario} \
 --enable-foreman-plugin-remote-execution \
---enable-foreman-proxy-plugin-remote-execution-script
+--enable-foreman-proxy-plugin-remote-execution-ssh
```

Comment:
While this is correct, I wonder why we tell users to do this at all.

Reply:
Okay, it seems to exist since https://bugzilla.redhat.com/show_bug.cgi?id=1610961, but that doesn't explain why it was added.

I'm voting dropping, as the feature will already be enabled in the backup just fine.

@ehelms ?